tests/tests.py

```diff
@@ -201,6 +201,68 @@ def test_elb(self):
                 'dev').config)
         self.assertEquals(known, config.elb())
 
+    def test_elb_with_reserved_chars(self):
+        from testfixtures import compare
+
+        self.maxDiff = None
```

Comment:
It wasn't long enough - this case had a diff of over 5000 characters when it wasn't right.